readme.md

```diff
@@ -32,6 +32,7 @@
 - [API quick reference and documentation](https://webgpu.rocks/) - WebGPU.rocks
 - [107 WebGPU Projects on GitHub](https://awesomeopensource.com/projects/webgpu) - AwesomeOpenSource.com
 - [r/WebGPU - Reddit](https://www.reddit.com/r/webgpu/) - WebGPU Subreddit
+- [webgpu.toys](https://webgpu.toys/) - Compute shader playground (like shadertoy)
```

Comment:
@munrocket the link is not working, it should be https://compute.toys/ ?

Reply:
Sorry my bad, you right.